app/server/app.js

```diff
@@ -72,128 +70,138 @@ app.get('/live-affluence', function(req, res){
     
 });
 
-// endpoint receiving the sms from twilio
-app.post('/twilio', function(req, res) {
+// // endpoint receiving the sms from twilio
+// app.post('/twilio', function(req, res) {
```

Comment:
Then just enable `/twilio` endpoint in `app`, and disable it in `admin` for now.
Losing the real-time capability in `admin` is not a big deal at the moment.
We'll address the `admin` real-time case later on with a dedicated reception server whose purpose will to be redirect incoming messages to `admin`or `app` depending on the context.